tests/pull.bats

```diff
@@ -249,3 +249,20 @@ load helpers
   umount /tmp/buildah-test
   rm -rf /tmp/buildah-test
 }
+
+@test "pull-policy" {
+  mkdir ${TESTDIR}/buildahtest
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy bogus alpine
+
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  run_buildah 125 inspect alpine
+
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json alpine
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy missing alpine
+  run_buildah inspect alpine
+
+  run_buildah rmi alpine
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy always alpine
```

Comment:
Same here. We need to go deeper and check the artifacts.

Reply:
I added some simple tests to make sure these are working.